Scripts/Linux/InstallFromSource.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# To test on a Debian-based Linux machine:
+#
+# ./InstallFromSource.sh
+# (outputs Git, GCM, and Scalar versions)
+#
+# Small clone should work:
+# scalar clone https://dev.azure.com/gvfs/ci/_git/ForTests
+#
+# Big clone for authenticated repo should work:
+# scalar clone <url>
+#
+# Note: if you are using a headless Linux machine, then GCM Core will
+# not be able to store your credentials. Instead, create a PAT and
+# use that for password (multiple times). This will be fixed soon.
+
+# Nuke previous data
+rm -rf git-vfs.deb gcm-core scalar log.txt
+
+echo "Installing dotnet SDK 3.1"
+yes | sudo apt-get install dotnet-sdk-3.1 dotnet-runtime-3.1
```

Comment:
I think this might benefit from some expansion ... first because want to ensure the Microsoft package signing key is in place per the [instructions](https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#2004-).  Maybe something like:
```
sudo apt-get update
sudo apt-get install -y apt-transport-https && sudo apt-get update

sudo apt-get install -y wget
wget https://packages.microsoft.com/config/ubuntu/$(lsb_release -rs)/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
sudo dpkg -i packages-microsoft-prod.deb

sudo apt-get install -y dotnet-sdk-3.1
```

You don't need the `dotnet-runtime-3.1`, I think, as it was installed automatically for me as a dependency of the SDK, and the [docs](https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#install-the-sdk) seem to indicate it's not required if the SDK is installed.

Reply:
And then, I guess, add `packages-microsoft-prod.deb` to the list of stuff to nuke in the `rm -rf` above.